### ISSUE TITLE
Option z of bmcdiscover was broken by PR5582

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#5617]
xcat version :  
```
 Version 2.14.4 (git commit 547f335aa3a72d4668bf7b100ddb03ac3f4128a1, built Thu Sep  6 10:17:11 UTC 2018)
```

The option `z` of `bmcdiscover` was broken by PR #5582. 

The man page are

```
NAME
       bmcdiscover - Discover Baseboard Management Controllers (BMCs) using a scan method
.....
      -z        List the data returned in xCAT stanza format
```

The original output was:
```
------START::bmcdiscover_range_z::Time:Tue Sep  4 09:21:28 2018------

RUN:bmcdiscover --range  50.6.36.1 -u ADMIN -p ADMIN -z [Tue Sep  4 09:21:28 2018]
ElapsedTime:1 sec
RETURN rc = 0
OUTPUT:
node-9006-22c-1309ala:
        objtype=node
        groups=all
        bmc=50.6.36.1
        cons=ipmi
        mgt=ipmi
        mtm=9006-22C
        serial=1309ALA
        bmcusername=ADMIN
        bmcpassword=ADMIN

CHECK:rc == 0   [Pass]
CHECK:output =~ bmc=50.6.36.1   [Pass]

------END::bmcdiscover_range_z::Passed::Time:Tue Sep  4 09:21:29 2018 ::Duration::1 sec------
```

After PR #5582, it was changed to 
```
------START::bmcdiscover_range_z::Time:Thu Sep  6 09:30:05 2018------

FILENAME:/opt/xcat/bin/../share/xcat/tools/autotest/testcase//bmcdiscover/cases0

RUN:bmcdiscover --range  50.6.36.1 -u ADMIN -p ADMIN -z [Thu Sep  6 09:30:05 2018]
ElapsedTime:1 sec
RETURN rc = 0
OUTPUT:
Found match node node-9006-22c-1309ala with bmc ip address: 50.6.36.1, rsetboot/rpower node-9006-22c-1309ala to continue hardware discovery.
CHECK:rc == 0	[Pass]
CHECK:output =~ bmc=50.6.36.1	[Failed]

------END::bmcdiscover_range_z::Failed::Time:Thu Sep  6 09:30:06 2018 ::Duration::1 sec------
```

**Comment by robin2008** [comment]
Fixed it in #5619

**Comment by xuweibj** [comment]
Verified in automation env. Close it.
